dev/sbtl_profile.py

```
@@ -0,0 +1,130 @@
+            if 0 < heos.Q() < 1: continue
+            h_arr[k] = heos.hmass(); valid[k] = True
+        except Exception:
+            # skip out-of-envelope timing point
+            pass
+    Tarr = Tarr[valid]; Parr = Parr[valid]; h_arr = h_arr[valid]
+    M = len(Tarr)
+    print(f'\n  {label}  (M={M} valid points)', flush=True)
+    for name, BCK in [('BICUBIC', bicu), ('SBTL', sbtl), ('HEOS', heos)]:
+        t0 = time.perf_counter()
+        for k in range(M):
+            try: BCK.update(CP.HmassP_INPUTS, h_arr[k], Parr[k])
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)